pyfstat/core.py

```diff
@@ -1043,7 +1049,7 @@ def _calculate_predict_fstat_cumulative(
                 maxStartTime=t,
                 sftfilepattern=self.sftfilepattern,
                 IFO=IFO,
-                **pfs_input
+                **pfs_input,
```

Comment:
I guess this and below is another case of "new black" getting randomly picky on unchanged lines...?

Reply:
Yes, it is.